crosswalks/ogcapi-coverages.md

```diff
@@ -0,0 +1,427 @@
+    + [Meta / API capabilities](#meta-api-capabilities)
+    + [Well-known URIs](#well-known-uris)
+    + [Landing page](#landing-page)
+    + [Conformance](#conformance)
+    + [API description](#api-description)
+    + [Authentication](#authentication)
+    + [Data discovery](#data-discovery)
+    + [Data description](#data-description)
+    + [Data access](#data-access)
+    + [Data subsetting](#data-subsetting)
+    + [Data scaling](#data-scaling)
+    + [Tiles](#tiles)
+    + [Media encoding](#media-encoding)
+  * [References](#references)
+
+
+## Introduction
+
+This *OGC API - Coverages* draft specification establishes how to access coverages as defined by the
+[OGC Abstract Specification Topic 6](https://portal.ogc.org/files/?artifact_id=19820) / 
+[ISO 19123](https://www.iso.org/standard/40121.html) *Schema for coverage geometry and functions*,
+and it thus represents an alternative to the established OGC
+[Web Coverage Service (WCS)](http://docs.opengeospatial.org/is/17-089r1/17-089r1.html) 
+interface standard (v2.1).
+
+The strategy behind the development of the *OGC API - Coverages* specification is to 
+develop a minimal, relatively simple API standard first, with new capabilities being incrementally
+added based on the community demands. Also, while defining how to access (portions of)
+coverages from a catalog, the standard does not specify the encoding of the coverage
+itself given in response, not limiting it to "internal" OGC encodings like the 
+[CIS 1.1](http://www.opengis.net/doc/IS/cis/1.1.1), or 
+[CF-netCDF](https://www.ogc.org/standard/netcdf/)), but also to e.g.
+[CoverageJSON](https://covjson.org/), [COPC](https://copc.io/), etc.
+
+This crosswalk will proceed by comparing such standard with the openEO API, touching
+all the functionalities that are declared in the OGC standard, i.e.:
+
+- (meta)data retrieval
+- subsetting through dimension(s)
+- range (bands) subsetting
+- up/down-scaling
+- data retrieval as tiles
+
+NOTE: the following abbreviations will be used throughout the document:
+
+- **openEO** : *openEO API, v1.2.0*
+- **OAC1** : *OGC API - Coverages - Part 1*
+
+## Nomenclature
+
+Despite stemming from the [OGC API - Features](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html) standards,
+openEO/STAC and OGC have both a different terminology and data models, so here
+we put the definitions of the most relevant terms of both "worlds".
+
+Most notably, an OGC [*coverage*](http://myogc.org/go/coveragesDWG) &mdash;
+even though formally defined as a *feature*
+(and not a *collection* thereof) &mdash; is frequently an abstraction for a 
+*collection* of (generally, but not limited to, gridded) datasets/items
+(like time-series of satellite images for instance),
+so we can generally compare it with STAC [collections](https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md),
+especially [datacubes](https://openeo.org/documentation/1.0/datacubes.html).
+
+### OGC
```

Comment:
For better understanding, it might help to give examples here so that we can more easily map between openEO and Coverages. 

I'm trying to map right now, but I'm unsure:
- collection = collection?
- coverage/feature = data cube?
- range = dimension?
- domain = dimension labels?

Reply:
The mapping is more like this:

- collection = collection
- coverage/feature = data cube / item / collection 
- range set = the actual data (pixels, voxels, n-xels?!)
- range type = the description of the data space (bands) = dimension::bands
- domain set = dimensions::spatial, dimensions::temporal, dimensions::other

Reply:
(anyway I'm a bit confused here by your mapping: STAC datacube can also be a STAC collection, right?)

Reply:
Well, I'd keep STAC aside for now, it's just the metadata, but doesn't really care in this context (I think) as we are just mapping the coverage and data cube concept.

Your list makes sense to me, I'm just confused about the range type. What happend with the range type if the data is non-spectral? And why is other a domain set then?

Reply:
Okay, I just think that dimensions of type other in openEO are often more a range rather than a domain. It could for example be a data cube x,y,t,indices where indices contains ndvi, evi, ndsi etc. Then I think it should be a range in WCPS, right?

Reply:
The indices should go in the coverage range, exactly. Though this is not always the case! I modeled 5D marrays for climate indices projections with "model" and "scenario" as additional "other" dimensions (then x,y,t) (see an example [here](http://saocompute.eurac.edu/rasdaman/ows?&SERVICE=WCS&VERSION=2.0.1&REQUEST=DescribeCoverage&COVERAGEID=amt_19712100_tnst_1x1km_year_cordexadj_qdm)): I would **not** want these "other" dimensions to go into a coverage range, but rather stay as dimensions (domain). In the range I might want to keep the values only, or maybe I could put value and uncertainty, etc.